diff -Naur a/src/input.c b/src/input.c
--- a/src/input.c	2018-07-09 16:21:30.000000000 +0530
+++ b/src/input.c	2019-04-19 16:41:16.499174200 +0530
@@ -29,6 +29,8 @@
 #include "loader-private.h"
 #include <stdlib.h>
 
+static const char* const key_unknown = "";
+
 struct wpe_input_key_mapper {
     struct wpe_input_key_mapper_interface* interface;
 };
@@ -50,20 +52,20 @@
 const char*
 wpe_input_identifier_for_key_event(struct wpe_input_key_mapper* key_mapper, struct wpe_input_keyboard_event* event)
 {
-    return key_mapper->interface->identifier_for_key_event(event);
+    return key_unknown;
 }
 
 __attribute__((visibility("default")))
 int
 wpe_input_windows_key_code_for_key_event(struct wpe_input_key_mapper* key_mapper, struct wpe_input_keyboard_event* event)
 {
-    return key_mapper->interface->windows_key_code_for_key_event(event);
+    return 0;
 }
 
 __attribute__((visibility("default")))
 const char*
 wpe_input_single_character_for_key_event(struct wpe_input_key_mapper* key_mapper, struct wpe_input_keyboard_event* event)
 {
-    return key_mapper->interface->single_character_for_key_event(event);
+    return key_unknown;
 
 }
