--- a/Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/CompositingCoordinator.cpp	2019-09-10 13:31:55.490859688 +0100
+++ b/Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/CompositingCoordinator.cpp	2019-09-10 13:30:59.067835322 +0100
@@ -412,7 +412,7 @@
     return *m_paintingEngine;
 }
 
-const Seconds releaseInactiveAtlasesTimerInterval { 500_ms };
+const Seconds releaseInactiveAtlasesTimerInterval { 10_s };
 
 void CompositingCoordinator::scheduleReleaseInactiveAtlases()
 {
